app/assets/stylesheets/base/_tables.scss

```diff
@@ -0,0 +1,42 @@
+@import "patterns/_borders";
```

Comment:
You don't need to import the individual files in each stylesheet. Just import all of them in `application.scss`